src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNumber = 100;
```

Comment:
Please check naming convention for constants in the course materials.
Also try think more accurate naming that express the meaning of the value that constant defines.